tools/delete_timepoint.php

```diff
@@ -173,19 +173,27 @@ function deleteTimepoint($CandID, $sessionID, $confirm, $printToSQL, $DB, $outpu
         ['sid' => $sessionID]
     );
 
+    $loris = new \LORIS\LorisInstance(
```

Comment:
This script already includes `generic_includes.php`, there's already a LorisInstance object instantiated there that should just be used

Reply:
Done